app/assets/stylesheets/show.scss

```diff
@@ -0,0 +1,65 @@
+.divider {
+  margin: 1.6rem;
+}
+
+.form-divider {
+  margin: 0;
+}
+
+.info-description {
+  border-top: 2px dotted $primary-color;
+  margin-top: 0.3rem;
+  padding-top: 0.3rem;
+  font-size: 0.8rem;
+}
+
+.info-name {
+  float: left;
+  color: $primary-color;
+  font-weight: bold;
+  font-size: 1.1rem;
+}
+
+.info-price {
+  float: right;
+  font-weight: bold;
+  margin-right: 0.5rem;
+}
+
+.info-quantity {
+  float: right;
+
+  [type="text"],
+  textarea {
+    width: 2rem;
+    height: 1.75rem;
+    margin-top: -0.2rem;
+    margin-right: 0;
+    margin-bottom: 0;
+    padding: 0;
+  }
+}
+
+.menu-title {
+  text-align: center;
+}
+
+.block-list ul {
+  margin: 1em;
```

Comment:
Properties should be ordered border, font-size, line-height, list-style-position, margin, margin-bottom